manifest.json

```diff
@@ -1,5 +1,809 @@
+                                  "countWords": true
+                                },
+                                "strong": {
+                                  "score": 0,
+                                  "countWords": false
+                                },
+                                "blockquote": {
+                                  "score": 0,
+                                  "countWords": false
+                                },
+                                "h1": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "h2": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "h3": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "h4": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "h5": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "h6": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "a": {
+                                  "score": 5,
+                                  "countWords": true
+                                },
+                                "li": {
+                                  "score": 0.5,
+                                  "countWords": true
+                                },
+                                "ul": {
+                                  "score": 1,
+                                  "countWords": true
+                                },
+                                "td": {
+                                  "score": 0,
+                                  "countWords": true
+                                },
+                                "hr": {
+                                  "score": 0,
+                                  "countWords": true
+                                },
+                                "pre": {
+                                  "score": 0,
+                                  "countWords": true
+                                },
+                                "ol": {
+                                  "score": 1,
+                                  "countWords": true
+                                }
+                              },
+                              "type": "object",
+                              "patternProperties": {
+                                "^(.*)$": {
```

Comment:
Counting the entire comment?

Reply:
No this defines how the properties of the object are called, I believe this is a `Record<string, object>`